src/side-sheet/stories/index.stories.js

```diff
@@ -193,3 +194,62 @@ storiesOf('side-sheet', module)
       </Manager>
     </Box>
   ))
+  .add('positions', () => (
```

Comment:
These stories are great – do we need to make any updates to the docs as well?

Reply:
Docs update would be awesome. However, I haven't updated docs for anything in v4 yet. I am open for doing this in one fell swoop as we ready v4. I am ok to create a new issue+PR for this.